src/main/java/org/opentestsystem/ap/ims/service/ItemService.java

```diff
@@ -35,9 +35,9 @@
 @Component
 public class ItemService {
 
-    private static final boolean NO_DO_NOT_COMMIT_ON_SAVE = false;
+    private static final boolean KEEP_CHANGES_PRIVATE = false;
```

Comment:
I plan on taking a look through the common and IMS code in the IDE in the next couple of sprints.  I'd love to meet with you at that point to provide feedback.  It's looking great from what I've seen in the pull requests.